packages/core/src/js/tracing/reactnativeprofiler.tsx

```diff
@@ -12,13 +12,24 @@ const ReactNativeProfilerGlobalState = {
   },
 };
 
+type ProfilerConstructorProps = ConstructorParameters<typeof Profiler>[0];
+type ReactNativeProfilerConstructorProps = ProfilerConstructorProps &
```

Comment:
We don't use the extra `removeUpdateProps` anymore.

Reply:
Good catch! I have removed it on https://github.com/getsentry/sentry-react-native/pull/4740/commits/cf5c90fa29082e70b0d1ac905f54cbc34e92e886